tests/components/sensor/test_rest.py

```diff
@@ -24,13 +25,6 @@ def tearDown(self):
         """Stop everything that was started."""
         self.hass.stop()
 
-    def test_setup_missing_config(self):
```

Comment:
That's fine with me, but just for my understanding, if a voluptuous required configuration is missing, it's never going to even enter setup_platform, right? Do we need to test this functionality in every platform?